tiflash: Update tiflash note of importing data by Lightning (#4516)
pingcap/docs: tiflash/use-tiflash.md

    CREATE TABLE table_name like t
    ```

> * For the version earlier than v4.0.6, if you create the TiFlash replica before using TiDB Lightning to import the data, the data import will fail. You must import data to the table before creating the TiFlash replica for the table.

Reviewer: ```suggestion
* For versions earlier than v4.0.6, if you create the TiFlash replica before using TiDB Lightning to import the data, the data import will fail. You must import data to the table before creating the TiFlash replica for the table.
```

> * If TiDB and TiDB Lightning are both v4.0.6 or later, no matter a table has TiFlash replica(s) or not, you can import data to that table using TiDB Lightning. It depends on the NIC bandwidth on the lightning host, the CPU and disk load of the TiFlash node, and the number of TiFlash replicas.

Reviewer: ```suggestion
* If TiDB and TiDB Lightning are both v4.0.6 or later, no matter a table has TiFlash replica(s) or not, you can import data to that table using TiDB Lightning. Note that this might slow the TiDB Lightning procedure, which depends on the NIC bandwidth on the lightning host, the CPU and disk load of the TiFlash node, and the number of TiFlash replicas.
```

> * If TiDB and TiDB Lightning are both v4.0.6 or later, you can create TiFlash replicas before importing data using TiDB Lightning. Note that this might slow the TiDB Lightning procedure. It depends on the NIC bandwidth on the lightning host, the CPU and disk load of the TiFlash node, and the number of TiFlash replicas.

Author: ```suggestion
* If TiDB and TiDB Lightning are both v4.0.6 or later, no matter a table has TiFlash replica(s) or not, you can import data to that table using TiDB Lightning. It depends on the NIC bandwidth on the lightning host, the CPU and disk load of the TiFlash node, and the number of TiFlash replicas.
```

> * For TiDB and TiDB Lightning version v4.0.6 or later, you can create TiFlash replicas before importing data by using TiDB Lightning. Note that this might slow the TiDB Lightning procedure. It depends on the bandwidth of the NIC on the lightning host, the CPU and disk load of the TiFlash node, and the number of TiFlash replica, etc.

Reviewer: ```suggestion
* If TiDB and TiDB Lightning are both v4.0.6 or later, you can create TiFlash replicas before importing data using TiDB Lightning. Note that this might slow the TiDB Lightning procedure. It depends on the NIC bandwidth on the lightning host, the CPU and disk load of the TiFlash node, and the number of TiFlash replicas.
```
